q2cli/tools.py

```diff
@@ -80,6 +98,9 @@ def show_importable_types(ctx, param, value):
               callback=show_importable_types, expose_value=False,
               help='Show the semantic types that can be supplied to --type '
                    'to import data into an artifact.')
+@click.option('--show-importable-formats', is_flag=True, is_eager=True,
+              callback=show_importable_formats, expose_value=False,
+              help='Show all possible source formats available for import.')
```

Comment:
Can you reference the `--source-format` parameter in this help text, that way users know what these formats can be plugged into? Something like: *Show formats that can be supplied to --source-format to import data into an artifact.*